postgres/pgdump.go

```diff
@@ -585,6 +620,10 @@ func updateSchema(conv *internal.Conv, table string, cs []constraint, stmtType s
 			ct := conv.SrcSchema[table]
 			ct.ForeignKeys = append(ct.ForeignKeys, toForeignKeys(c)) // Append to previous foreign keys.
 			conv.SrcSchema[table] = ct
+		case nodes.CONSTR_UNIQUE:
+			ct := conv.SrcSchema[table]
+			ct.Indexes = append(ct.Indexes, schema.Index{Name: "", Unique: true, Keys: toSchemaKeys(conv, table, c.cols)})
+			conv.SrcSchema[table] = ct
```

Comment:
I wonder why do we need `conv.SrcSchema[table] = ct` here (and similarly in the existing code in the cases above since points to the element in the map and isn't copied by value simply)

Reply:
as ct is not a pointer, after changing actual content inside ct, it doesn't actually update conv.SrcSchema[table], so we have to reasign it to conv.SrcSchema[table].